lib/amo-client.js

```diff
@@ -211,13 +219,23 @@ Client.prototype.downloadSignedFiles = function(signedFiles, options) {
     });
   }
 
-  showProgress();
-  signedFiles.forEach(function(file) {
-    allDownloads.push(download(file.download_url));
-  });
+  // TODO: handle 404 downloads
```

Comment:
I'll file a separate issue for this but it probably only impacts local development